2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -50,39 +26,21 @@ library(here)
 replicates <- 1:16
 #here() is used above to make the filepath start from the correct location (the top of the project directory)
 #Sys.glob returns all the files matching the above pattern, which should be all peptide groups files
-pep_infiles <- Sys.glob(here('inst/extdata/*_PeptideGroups.txt'))
-psm_infiles <- Sys.glob(here('inst/extdata/*_PSMs.txt'))
```

Comment:
Was being defined again later as well. Better just to define `psm_infiles` where you use it (later on)